pallets/common/src/lib.rs

```diff
@@ -14,8 +14,45 @@
 // You should have received a copy of the GNU General Public License
 // along with Unique Network. If not, see <http://www.gnu.org/licenses/>.
 
+//! # Common pallet
+//!
+//! The Common pallet provides functionality for handling collections.
+//!
+//! ## Overview
+//!
+//! The Common pallet provides functions for:
+//!
+//! - Setting and approving collection soponsor.
+//! - Get\set\delete allow list.
+//! - Get\set\delete collection properties.
+//! - Get\set\delete collection property permissions.
+//! - Get\set\delete token property permissions.
+//! - Get\set\delete collection administrators.
+//! - Checking access permissions.
+//! - Provides an interface for common collection operations for different collection types.
+//! - Provides dispatching for implementations of common collection operations, see [dispatch] module.
+//! - Provides functionality of collection into evm, see [erc] and [eth] module.
+//!
+//! ### Terminology
+//! **Collection sponsor** - For the collection, you can set a sponsor, at whose expense it will
+//! be possible to mint tokens.
+//!
+//! **Allow list** - List of users who have the right to minting tokens.
+//!
+//! **Collection properties** - Collection properties are simply key-value stores where various
+//! metadata can be placed.
+//!
+//! **Collection property permissions** - For each property in the collection can be set permission
+//! to change, see [PropertyPermission].
+//!
+//! **Permissions on token properties** - Similar to _permissions on collection properties_,
+//! only restrictions apply to token properties.
+//!
+//! **Collection administrator** - For a collection, you can set administrators who have the right
+//! to most actions on the collection.
+
+#![warn(missing_docs)]
```

Comment:
Actually, the documentation doesn't compile because of this -- many missing documentation for struct fields and function arguments. You can use the `cargo doc` command and check the console for warnings within the common pallet, and use the `Rust Doc Viewer` extension to verify that everything works.

Reply:
`cargo doc ` will not show warnings without `#![warn(missing_docs)]` and code compiles well with this.

Reply:
It will only not show missing documentation. Errors are still shown. Anyway, as for documentation not generating, it seems to be a problem with `Rust Doc Viewer`, for some reason, with or without it. I managed to open it from the generated doc for another crate, so it's fine.